docs/release/trg-1/trg-1-2.md

```diff
@@ -2,15 +2,69 @@
 title: TRG 1.02 - INSTALL.md
 ---
 
-| Status | Created      | Post-History |
-|--------|--------------|--------------|
-| Active | 07-Mar-2023  |              |
-| Draft  | 13-Sept-2022 |              |
+| Status | Created      | Post-History                                        |
+|--------|--------------|-----------------------------------------------------|
+| Update | 13-Dez-2023  | add best practices and an example, rephrase the why |
+| Active | 07-Mar-2023  |                                                     |
+| Draft  | 13-Sept-2022 |                                                     |
 
 ## Why
 
-If a repository contains code which needs to be configured/setup or installed, the `README.md` file should cover this information. If this is too much text, it can be moved into a `INSTALL.md` file.
+If a repository contains code which needs to be configured/setup or installed, the `README.md` file should cover this information.
+
+If these instructions are too long, they can be moved into a dedicated `INSTALL.md` file.
+
+Having a proper `INSTALL.md` file is crucial for ensuring a seamless and user-friendly experience for the product-users.
+
+In conclusion, a proper `INSTALL.md` file should be mainly adapted to end-users, but also developers. It is not a technical necessity.
+It's a strategic investment in user satisfaction, community engagement, and the overall success of the product. It reflects a commitment to inclusivity and usability, ensuring that a diverse user base can effortlessly install and enjoy the benefits of your product.
 
 ## Description
 
 Create a `INSTALL.md` file in the main folder of the repository and add all installation relevant information.
+
+## Best Practices
+
+a well structured installation guide includes
```

Comment:
Start sentence with capital letter and fix markdown error as list did not have blank line before itself.

```suggestion
A well structured installation guide includes

```

Reply:
https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/559/commits/9d85c5b5db5e8001ac155a3373d25b195300ffcf